src/Rapidez.php

```diff
@@ -61,4 +62,21 @@ public function fancyMagentoSyntaxDecoder(string $encodedString): object
 
         return json_decode(str_replace(array_values($mapping), array_keys($mapping), $encodedString));
     }
+
+    public function getStores(): Collection
+    {
+        $storeModel = config('rapidez.models.store');
+
+        return $storeModel::all();
```

Comment:
I'm guessing it is potentially a good idea to wrap this in an array cache in case this gets executed multiple times during the same run, so we don't need to fetch the store models each time.
```suggestion
        return cache()->store('array')->rememberForever('allStores', fn() => (config('rapidez.models.store'))::all());
```

Reply:
They are probably already cached from here: https://github.com/rapidez/core/blob/8732ca140a3b899830902686531a30c66a73b64b/src/Models/Store.php#L28. Maybe create a `getCached()` there so it's also possible to get all stores.